packager/packager.js

```diff
@@ -150,6 +150,19 @@ function getDevToolsLauncher(options) {
   };
 }
 
+// A status page so the React/project.pbxproj build script
+// can verify that packager is running on 8081 and not 
+// another program / service.
+function statusPageMiddleware (req, res, next) {
```

Comment:
nit: no space between function name and opening paren:

  function statusPageMiddleware(req, res, next) {